### ISSUE TITLE
App elements with highlight focus color that's the same as their base color while in black theme

### ISSUE THREAD

**Issue by sudomain** [ankidroid/Anki-Android#19343]
### Checked for duplicates?

- [x] This issue is not a duplicate

### Does it also happen in the desktop version?

- [x] This bug does not occur in the latest version of Anki Desktop

### What are the steps to reproduce this bug?

Originally described in #19320 (number 3):

> When the app theme is "black", There are a few app components that can gain focus but the focus highlight color is also black or a shade that is not very different from the base color. This results in no (or a very muted) visual feedback for the user. Examples of these components include the cards/notes in the card browser, the popup that shows after clicking Change display order in the card browser, and the pop that shows after clicking Import in the deck picker.

1. Connect a USB or BT keyboard to your phone or use an app such as Key Mapper that can send KEYCODE_TAB
2. Switch Ankidroid to black theme
3. Press TAB a few time to try to highlight/focus on the cards/notes in the card browser, the popup that shows after clicking Change display order in the card browser, or the popup that shows after clicking Import in the deck picker.
4. Switch to a different theme
5. Repeat step 3 and note that these app elements can gain focus in the black theme, however the focus highlight color is the same color thus there's no visual feedback.

### Expected behaviour

A different color for the affected items when they gain focus while in black theme.

### Debug info

```text
AnkiDroid Version = 2.22.3 (4f2ec68)
Backend Version = 0.1.60-anki25.07.5 (25.07.5 7172b2d26684c7ef9d10e249bd43dc5bf73ae00c)
Android Version = 12 (SDK 31)
ProductFlavor = full
Device Info = samsung | samsung | r0q | r0qsqw | SM-S901U | qcom
Webview User Agent = Mozilla/5.0 (Linux; Android 12; SM-S901U Build/SP1A.210812.016; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/140.0.7339.208 Mobile Safari/537.36
ACRA UUID = 2c997926-de3d-48b5-a09f-9b712962622a
FSRS = 4.1.1 (Enabled: false)
Crash Reports Enabled = true
```

### Research

- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by sakesane** [comment]
I find a related pr which claimed to "have fixed part of issue #19343", but this issue is still open.

Are there any problems remained to be fix?

**Comment by snowtimeglass** [comment]
@sakesane 

The issue of the following part has been fixed in #19343, I think.
> Press TAB a few time to try to highlight/focus on the cards/notes in the card browser, 

 _Originally posted by @sudomain  in [#19343](https://github.com/ankidroid/Anki-Android/issues/19343)_

- #19343

But I have no idea about the following parts (as I haven't been able to reproduce these popups): 

> the popup that shows after clicking Change display order in the card browser, or the popup that shows after clicking Import in the deck picker.

**Comment by sudomain** [comment]
"Popup" may not have been the correct word (if anyone knows what is, let me know) as my app component lingo isn't the best. What I meant was this:

![Image](https://github.com/user-attachments/assets/b37f2377-8e5e-4bb4-bfd4-f5d605ac3f12)

Though the popup isn't black itself, while in the black theme the focus highlight color is the same shade of gray as the popup background color.

Also, it's very odd, but I also struggled to reproduce this just now as I can get this menu to gain focus on some devices sending TAB, but not on other devices also sending TAB, but that's a separate issue.

**Comment by sudomain** [comment]
My op was not worded in the clearest way. I edited it for some clarity and to note that the first item had a PR merged that fixed it.
